### PR TITLE
Redo packaging structure to match other projects.

### PR DESCRIPTION
**Pull request by jguiditta**
This project has not been updated in a while so it needs a
few patches to bring it in line with our other projects.

Signed-off-by: Jason Guiditta <jguiditt@redhat.com>

### CODE REVIEW

**Review by fuzzball81 — APPROVED** [review]

**Comment by fuzzball81** [comment]
Looks like we need to add the 2 approvers rule here. Sorry @jpichon I wanted you to have a chance to look this over as well.

File: Makefile [hunk 1]
```diff
@@ -52,38 +52,31 @@ clean-test: ## remove test and coverage artifacts
...
-	pipenv run coverage report -m
-	pipenv run coverage html
-	$(BROWSER) htmlcov/index.html
+	pipenv run  tox -epy
 
 docs: ## generate Sphinx HTML documentation, including API docs
-	rm -f docs/koji_wrapper.rst
-	rm -f docs/modules.rst
-	pipenv run sphinx-apidoc -o docs/ koji_wrapper
+	pipenv run sphinx-apidoc -o docs/ koji_wrapper ##TODO: remove
```
Comment (jpichon) [inline Makefile:67]: Any reason for the TODO?

File: Makefile [hunk 2]
```diff
@@ -95,7 +88,6 @@ install: init ## install the package to the active Python's site-packages
 	pipenv run python setup.py install
 
 dev: init ## set up a development environment
```
Comment (jpichon) [inline Makefile:90]: The suggestion was to use 'clean' for the dev target recently (https://github.com/release-depot/toolchest/pull/43)

File: README.rst [hunk 3]
```diff
@@ -27,13 +27,16 @@ This library only supports python 3. Some features may still work with python
 Development
 -----------
 
-There are several dependencies needed to build and work on koji_wrapper.  Using
-your distribution's package manager, install these system packages::
+There are several dependencies needed to build and work on koji_wrapper.  These
+are a result of a dependency upstream from us in the `cryptography`_ library,
+which lists the packages needed if you run into problems pip installing our
+requirements.
```
Comment (jpichon) [inline README.rst:33]: Nice explanation!

**Review by jpichon — COMMENTED** [review]
LGTM to me, too!

File: Makefile (hunk 1 above)
Comment (jguiditta) [reply Makefile:67]: I guess I could have left that out of the commit, the change is happening in the upcoming commit where I update the docs to match other projects.

**Comment by jguiditta** [comment]
> Looks like we need to add the 2 approvers rule here. Sorry @jpichon I wanted you to have a chance to look this over as well.

I probably should have done a mergify PR first, but I was intending to keep that change separate since it would not automerge.  I will push up a PR before my next actual changeset for structure.

**Pull request merged**